### ISSUE TITLE
New vs. Previous RF model

### ISSUE THREAD

**Issue by cmahony** [whmacken/CCISS_JournalPaper#2]
@whmacken, 
here is a comparison of the old vs. new RF model. 

OLD MODEL: 

![CCISS BGCEDA PredError SizeBias](https://user-images.githubusercontent.com/17104005/55656316-53e43680-57ab-11e9-972a-e194cb7f268b.png)

NEW MODEL: 
![CCISS BGCEDA PredError SizeBias](https://user-images.githubusercontent.com/17104005/55656401-a0c80d00-57ab-11e9-98d5-e7d196fe7666.png)

The differences are subtle. the new model is still underpredicting (shrinking) small units and overpredicting large units (top left panel). we had previously identified this as a problem of the balance of the training sample, so there still may be some tweaking to be done there. 

thoughts?

**Comment by whmacken** [comment]
I can do a fully balanced data set. That will up the small units considerably

Sent from my iPhone

On Apr 5, 2019, at 14:08, cmahony <notifications@github.com<mailto:notifications@github.com>> wrote:


@whmacken<https://github.com/whmacken>,
here is a comparison of the old vs. new RF model.

OLD MODEL:

[CCISS BGCEDA PredError SizeBias]<https://user-images.githubusercontent.com/17104005/55656316-53e43680-57ab-11e9-972a-e194cb7f268b.png>

NEW MODEL:
[CCISS BGCEDA PredError SizeBias]<https://user-images.githubusercontent.com/17104005/55656401-a0c80d00-57ab-11e9-98d5-e7d196fe7666.png>

The differences are subtle. the new model is still underpredicting (shrinking) small units and overpredicting large units (top left panel). we had previously identified this as a problem of the balance of the training sample, so there still may be some tweaking to be done there.

thoughts?

**Comment by whmacken** [comment]
I’m not really getting the same results, Colin. Generally the larger units are being underpredicted and the smaller BGCs over-predicted. The error ratio is a bit misleading in that it is easier to predict a large percent difference when the starting point count is small. Instead I simply plotted a scatterplot of mapped to predicted for the gridded data. Dots above the line are predicted more than is actual and those below under predicted. (I removed the MGPun since it is so much more abundance – it falls farthest below the line.
This relationship is more what I expected – I have over emphasized the smaller units relative to the larger ones.  Could probably scale back on the training point balancing  – however that has an effect on the model-ability of those units and the misclassification rate rises.
[cid:image001.jpg@01D4EBC8.779C8540]

**Comment by cmahony** [comment]
will, i can't see the image. can you reply in github? 
c

**Comment by whmacken** [comment]
I’m not really getting the same results, Colin. Generally the larger units are being underpredicted and the smaller BGCs over-predicted. The error ratio is a bit misleading in that it is easier to predict a large percent difference when the starting point count is small. Instead I simply plotted a scatterplot of mapped to predicted for the gridded data. Dots above the line are predicted more than is actual and those below under predicted. (I removed the MGPun since it is so much more abundance – it falls farthest below the line.
This relationship is more what I expected – I have over emphasized the smaller units relative to the larger ones.  Could probably scale back on the training point balancing  – however that has an effect on the model-ability of those units and the misclassification rate rises.

![image](https://user-images.githubusercontent.com/34073318/55660752-c873a100-57bc-11e9-816f-2f38a55a468a.png)
